packages/web-app-importer/src/extensions.ts

```diff
@@ -112,6 +112,7 @@ export const extensions = ({ applicationConfig }: ApplicationSetupOptions) => {
         {
           id: 'com.github.owncloud.web.import-file',
           type: 'action',
+          scopes: ['files', 'upload-menu'],
```

Comment:
do we already have any semantics for the scopes? 🙈 is `files` referring to `resources` or `files app`?

Reply:
Not yet 😅

> is `files` referring to `resources` or `files app`?

Hard to say tbh. Both, kinda 😄